### ISSUE TITLE
Error: The terraform-provider-tabular_v0.0.23 plugin crashed!

### ISSUE THREAD

**Issue by Quixotical** [tabular-io/terraform-provider-tabular#38]
When attempting to import a tabular database to my terraform state, I received the following error:

```
tabular_database.this[0]: Importing from ID "***********"...

╷
│ Error: Plugin did not respond
│ 
│ The plugin encountered an error, and failed to respond to the
│ plugin6.(*GRPCProvider).ImportResourceState call. The plugin logs may
│ contain more details.
╵

Stack trace from the terraform-provider-tabular_v0.0.23 plugin:

panic: runtime error: index out of range [1] with length 1

goroutine 52 [running]:
github.com/tabular-io/terraform-provider-tabular/internal/provider.(*databaseResource).ImportState(0xd6a8a0?, {0xd6a8a0, 0xc0003416b0}, {{0xc000134240?, 0xc75dc0?}}, 0xc00038aa00)
	github.com/tabular-io/terraform-provider-tabular/internal/provider/database.go:104 +0x1f1
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ImportResourceState(0xc0000f9600, {0xd6a8a0, 0xc0003416b0}, 0xc000384b40, 0xc000613628)
	github.com/hashicorp/terraform-plugin-framework@v1.1.1/internal/fwserver/server_importresourcestate.go:104 +0x462
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ImportResourceState(0xc0000f9600, {0xd6a8a0?, 0xc000341560?}, 0xc000390340)
	github.com/hashicorp/terraform-plugin-framework@v1.1.1/internal/proto6server/server_importresourcestate.go:44 +0x30b
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ImportResourceState(0xc0001aebe0, {0xd6a8a0?, 0xc00039dd10?}, 0xc00038a7d0)
	github.com/hashicorp/terraform-plugin-go@v0.18.0/tfprotov6/tf6server/server.go:849 +0x1f6
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ImportResourceState_Handler({0xc27500?, 0xc0001aebe0}, {0xd6a8a0, 0xc00039dd10}, 0xc000394d90, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.18.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:440 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0001dc1e0, {0xd6f058, 0xc00047e000}, 0xc0000dd680, 0xc000340450, 0x122a3b8, 0x0)
	google.golang.org/grpc@v1.56.1/server.go:1337 +0xdf3
google.golang.org/grpc.(*Server).handleStream(0xc0001dc1e0, {0xd6f058, 0xc00047e000}, 0xc0000dd680, 0x0)
	google.golang.org/grpc@v1.56.1/server.go:1714 +0xa36
google.golang.org/grpc.(*Server).serveStreams.func1.1()
	google.golang.org/grpc@v1.56.1/server.go:959 +0x98
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.56.1/server.go:957 +0x18c

Error: The terraform-provider-tabular_v0.0.23 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

I had previously done an import of a warehouse and had no issue, so this error surprised me. Since it says

> This is always indicative of a bug within the plugin

I decided to create an issue here to hopefully get it resolved

**Comment by smikloska** [comment]
Confirming 0.0.17 working when using the database name, the bug was introduced >= 0.0.18 with https://github.com/tabular-io/terraform-provider-tabular/pull/26/

**Comment by smikloska** [comment]
Fixed in v0.0.25
